example/gardener-local/kind/cluster/templates/cluster.yaml

```diff
@@ -1,5 +1,7 @@
 apiVersion: kind.x-k8s.io/v1alpha4
 kind: Cluster
+featureGates:
+  ExpandedDNSConfig: true
```

Comment:
Why is this needed? I'm just wondering because it is not mentioned anywhere.

Reply:
Multi-zone upgrade tests are failing due to this https://github.com/gardener/gardener/issues/7297, our e2e kind cluster   version is `v1.24.7`, so we must enable featureGate `ExpandedDNSConfig`

Reply:
Can you add a comment to it so that we don't forget about it?